spec/requests/api/v1/users/create_request_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+# See spec/support/requests_shared_examples.rb for shared examples
+describe 'Users API', type: :request do
+  describe 'POST /api/v1/users' do
+    let(:valid_attributes) do
+      {
+        email: Faker::Internet.email,
+        google_id: Faker::Alphanumeric.alphanumeric(number: 10),
+        google_image_url: Faker::Avatar.image,
+        zip_code: Faker::Address.zip_code.first(5),
+        summary: Faker::GreekPhilosophers.quote,
+        goal: rand(0..5),
+        availability_morning: [true, false].sample,
```

Comment:
Good use of `.sample` instead of just setting all values to either true or false